### PR TITLE
SILA-1627 - deliver vaccounts changes into master postman collection

### PR DESCRIPTION
**Pull request by johndburk**
Deliver virtual account changes into master postman collection

* https://sila.atlassian.net/browse/SILA-1627 - original vAcct epic
* https://sila.atlassian.net/browse/SILA-3695 - vAcct disputes epic

### CODE REVIEW

File: postman/Sila API v0.2 - Local Signer Server Edition.postman_collection.json [hunk 1]
```diff
@@ -5545,6 +5630,437 @@
 				}
 			]
 		},
+		{
+			"name": "Virtual Accounts",
+			"item": [
+				{
+					"name": "Open Virtual Account",
+					"request": {
```
Comment (pyscape) [inline postman/Sila API v0.2 - Local Signer Server Edition.postman_collection.json:5638]: It looks like there's already a `Virtual Accounts` folder in this collection. Is that intentional?

**Review by pyscape — CHANGES_REQUESTED** [review]
Looks like there's either a naming conflict in folders or a second folder got added by accident.

File: postman/Sila API v0.2 - Local Signer Server Edition.postman_collection.json (hunk 1 above)
Comment (johndburk) [reply postman/Sila API v0.2 - Local Signer Server Edition.postman_collection.json:5638]: ah, likely not.  let me fix

Comment (johndburk) [reply postman/Sila API v0.2 - Local Signer Server Edition.postman_collection.json:5638]: it's a dupe from a merge :-/  what a mess

**Pull request merged**